.github/workflows/ci.yml

```diff
@@ -14,6 +28,16 @@ jobs:
     runs-on: self-hosted
     steps:
     - uses: actions/checkout@v2
+    - name: License Check
+      run: |
+        set -x
+        git init
+        git add --all
+        git -c user.name="CI Bot" -c user.email="<>" commit -m "initial state"
+        addlicense ./
+        if [[ -n $(git status -s) ]]; then
```

Comment:
you can also do simply do this: `test -z "$(git status -s | tee /dev/stderr)"`

right now you're not printing which files have a diff.

Reply:
Actually the results are printed, since the `set -x` flag is on. I think `git status -s` is preferable to `git diff` because we want to see the list of the files, not the full license content for each file

Reply:
sounds good. you can still remove this in this case by doing:

git init && git add --all && addlicense . && test -z $(**git diff --name-only** | tee /dev/stderr)

Reply:
Personally I find that a bit less readable, but I made the change